translations/translations.ts

```diff
@@ -0,0 +1,9 @@
+import zh_hans from './languages/zh-Hans'
```

Comment:
You should move this file to `translations/languages/index.ts` as it's scoped to the languages directory, all imports are from that directory.